sw/device/silicon_creator/rom/rom_epmp_test.c

```diff
@@ -325,7 +327,9 @@ static void test_unlock_exec_eflash(void) {
 
   // Verify that execution within the region succeeds.
   // The image must consist of `unimp` instructions so that an illegal
-  // instruction exception is generated.
+  // instruction exception is generated. Because the region is not written and
+  // tests begin with the flash erased, this instruction is expected to be
+  // UINT32_MAX.
   CHECK(image[0] == kUnimpInstruction);
```

Comment:
Previously, nothing ever guaranteed that this would be true, heh. Now we just assume flash was never written, since this image isn't large enough to cover the addresses inspected.